themes-default/slim/views/vue-components/asset.mako

```diff
@@ -22,16 +22,17 @@ Vue.component('asset', {
     },
     data() {
         return {
-            isVisible: false
+            isVisible: false,
+            error: false
         };
     },
     computed: {
         src() {
-            const {seriesSlug, type, isVisible} = this;
+            const { error, seriesSlug, type, isVisible } = this;
             const apiRoot = document.getElementsByTagName('body')[0].getAttribute('api-root');
             const apiKey = document.getElementsByTagName('body')[0].getAttribute('api-key');
 
-            if (!isVisible || !seriesSlug || !type) {
+            if (error || !isVisible || !seriesSlug || !type) {
```

Comment:
This may actually cause a infinite loop if the default image is throwing a 404 too:
404 -> `error = true` -> `src = this.default` -> 404 -> `error = true` -> `src = this.default` ...

Reply:
If `this.default` is an error then wouldn't that mean the server is down? Any ideas on how we can get around this?

Reply:
We could include a base64 image for this so we wouldn't need to default back to a URL based one?

Reply:
> If this.default is an error then wouldn't that mean the server is down?

Either that, or the url in default was incorrect.
> Any ideas on how we can get around this?

Error count instead of a boolean?
@error="error++;"
if (error == 2)

Reply:
I just verified, there's no issue with that.
Seems like Vue is handling this correctly - If the image from the API fails it falls back to `this.default` and if `this.default` fails, nothing happens, since the value of `error` didn't change from `true`.